manifests/devices.pp

```diff
@@ -47,20 +47,21 @@
 
   if ( (versioncmp($::clientversion, '4.9.0') >= 0) and (! defined('$::serverversion') or versioncmp($::serverversion, '4.9.0') >= 0) ) {
     $hiera_devices = lookup('device_manager::devices', Hash, 'hash', {})
+    $hiera_defaults = lookup('device_manager::defaults', Hash, 'hash', {})
```

Comment:
If we anticipate having class-level defaults then they should use `device_manager::defaults` and devices should use something like `device_manager::device_defaults`.